react/react1/week1/todo-app/src/components/Subheading.jsx

```diff
@@ -0,0 +1,13 @@
+const Subheading = ({todos}) => {
+    const count = todos.length;
+    let subheading;
+    if (count > 0) {
+        const noun = count > 1 ? "tasks" : "task";
+        subheading = count + " " + noun;
+    }
```

Comment:
I would highly advise to move this logic to the parent component, so that Subheading only consumes a "subheading" props value